dnfdaemon-client/commands/install/install.cpp

```diff
@@ -85,7 +76,9 @@ void CmdInstall::run(Context & ctx) {
     }
 
     dnfdaemon::KeyValueMap options = {};
-    options["strict"] = strict_option->get_value();
+    if (ctx.skip_broken->get_value() == true) {
```

Comment:
`skip_broken` can be `nullptr`.

Reply:
It is `bool`. See https://github.com/rpm-software-management/libdnf/blob/14922825c78423af9e3fc5d0d0f5384b74f6cf62/include/libdnf/conf/option_bool.hpp#L80

Reply:
Oh, you meant `skip_broken` itself, not it's value. No, it is set to OptoinBool instance in `Context` class declaration.